pkg/target/target.go

```diff
@@ -326,7 +332,8 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 		Type: "object",
 		Properties: map[string]apiextensions.JSONSchemaProps{
 			"kinds": {
-				Type: "array",
+				Type:        "array",
+				Description: "kinds accepts a list of objects with apiGroups and kinds fields that list the groups/kinds of objects to which the constraint will apply. If multiple groups/kinds objects are specified, only one match is needed for the resource to be in scope.",
```

Comment:
No need to get into the structure of the resource, the json schema does that. Instead, add a `Description` entry for each sub field that describes its purpose.

Reply:
Updated.  PTAL